test/fw/ptl/utils/plugins/ptl_test_data.py

```diff
@@ -150,23 +150,26 @@ def __save_home(self, test, status, err=None):
             f.close()
             return
 
-        server = getattr(_test, 'server', None)
+        servers = getattr(_test, 'servers', None)
         moms = getattr(_test, 'moms', None)
         comms = getattr(_test, 'comms', None)
-        client = getattr(_test.server, 'client', None)
+        client = getattr(_test.servers, 'client', None)
         add_hosts = []
+        if len(servers) > 1:
```

Comment:
I think you should add only servers[1:] values to add_hosts as you are already capturing the servers[0] host via -H